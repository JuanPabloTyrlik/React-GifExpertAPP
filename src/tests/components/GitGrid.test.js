import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import GifGrid from '../../components/GifGrid';

describe('Tests on GifGrid', () => {
    const category = 'One Punch Man';
    const wrapper = shallow(<GifGrid category={category} />);
    test('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
