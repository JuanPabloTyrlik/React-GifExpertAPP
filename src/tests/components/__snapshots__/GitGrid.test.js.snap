// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on GifGrid should render correctly 1`] = `
<Fragment>
  <h3>
    One Punch Man
  </h3>
  <p
    className="animate__animated animate__flash"
  >
    Loading...
  </p>
  <div
    className="card-grid"
  />
</Fragment>
`;
